### PR TITLE
(#2894) Bugfix for two column flex rows on mini-landing.

### LINKED ISSUE #2894: Two-Item Feature Card Row with one item needs correct styling
### Issue description
When a two-item feature card row is added to a mini-landing page, and only one item is added to it, that item should look the same as cards in a row with two cards, and should sit flush left. 

**Works on microsites: https://www.cancer.gov/pediatric-adult-rare-tumor/rare-tumors** 

> **ESTIMATE** 20

### Steps to reproduce the issue

1. Add a two-item feature card row to a mini-landing page contents
2. Select an item for the first internal feature card
3. Do not add another card
4. Publish page

### What's the expected result?
- The card appears on the page at the dimensions of a two-item feature card, sitting in the left column, and flexes to be full width on mobile. 

### What's the actual result?
- Card appears in the smaller, inline feature card format, sitting in the middle of the row.

It does appropriately flex on mobile. 

### Additional details / screenshot
Content unpublished in CMS: https://www-cms.cancer.gov/research/annual-plan/stories

image

image

### PR DESCRIPTION
**Pull request by adriancofie**
ODE: http://ncigovcdode383.prod.acquia-sites.com
Closes #2894

### CODE REVIEW

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/minilanding/Minilanding.scss [hunk 1]
```diff
@@ -15,6 +15,11 @@
   }
 }
 
+div.row.feature-primary.flex-columns {
+  display: -webkit-box;
```
Comment (arcepaul) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/minilanding/Minilanding.scss:19]: Can probably do without this? Autoprefixer should take care of this.

Comment (adriancofie) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/minilanding/Minilanding.scss]: Checked, you're right it's redundant

Comment (adriancofie) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/minilanding/Minilanding.scss]: Looks good on the ODE:  http://ncigovcdode383.prod.acquia-sites.com/about-nci/organization/ccct

**Comment by adriancofie** [comment]
http://ncigovcdode383.prod.acquia-sites.com/about-nci/organization/ccct

**Pull request merged**